src/sdk/map_api_types.js

```diff
@@ -174,6 +177,10 @@ class ChunkAPI {
         };
     }
 
+    _encrypt_cipher_key(cipher_key, master_key_id) {
+        if (!master_key_id) return cipher_key;
+        return this.system_store.master_key_manager.encrypt_buffer_with_master_key_id(from_b64(cipher_key), master_key_id).toString('base64');
```

Comment:
Why do we need to do 2 conversions.
toString('base64') and then fromBase64().

Reply:
fixed, it was a minor bug when master_key_id is undefined 👍